packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/TaskNotificationEventTypes.java

```diff
@@ -14,6 +16,9 @@ public class TaskNotificationEventTypes {
 
 	public static ClientEventType getTypeFor(String taskName) {
 		final ClientEventType eventType = clientEventTypes.get(taskName);
+		if (eventType == null) {
+			log.warn("Event type not found for task: " + taskName);
+		}
 		return eventType == null ? ClientEventType.TASK_UNDEFINED_EVENT : eventType;
```

Comment:
While you do the test, include the next line within 😄 
```suggestion
		if (eventType == null) {
			log.warn("Event type not found for task: " + taskName);
			return ClientEventType.TASK_UNDEFINED_EVENT
		}
		return eventType;
```

Reply:
what a thinker